packages/web/src/scss/components/Button/index.html

```diff
@@ -4,13 +4,13 @@
 
     <h2 class="docs-Heading">Variants</h2>
 
-    <button type="button" class="Button Button--primary">Primary button</button>
-    <button type="button" class="Button Button--secondary">Secondary button</button>
-    <button type="button" class="Button Button--tertiary">Tertiary button</button>
-    <button type="button" class="Button Button--danger">Danger button</button>
+    <button type="button" class="Button Button--primary Button--medium">Primary button</button>
```

Comment:
This means it's a breaking change: My `Button`s will be broken until I add a size class.